CMakeLists.txt

```diff
@@ -19,6 +19,11 @@ endif()
 
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+# Provide -stdlib=libc++ if necessary for cstdint to be defined
+if(CMAKE_CXX_LIBCPP)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -stdlib=libc++")
```

Comment:
Why not just pass -stdlib=libc++ as one of the CXX_FLAGS from the mci script?  Just wondering if this is overkill.